roles/bind-keycloak-apb/templates/client_configmap.j2

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ keycloak_configmap_name }}-{{ consumer }}
+  namespace: {{ namespace }}
+  labels:
+    mobile: enabled
+    serviceName: keycloak
+    consumerID: {{ consumer }}
+data:
```

Comment:
needs `clientId: {{CLIENT_ID}}`

Reply:
I can change that if that's what's expected on the UI side, but me and @sedroche preferred `consumerID` as it works for both clients and services. wdyt?